contracts/src/chainlink/solidity/emergency/StarkNetValidator.sol

```diff
@@ -74,15 +122,152 @@ contract StarkNetValidator is TypeAndVersionInterface, AggregatorValidatorInterf
     uint256, /* currentRoundId */
     int256 currentAnswer
   ) external override checkAccess returns (bool) {
-    bool status = currentAnswer == ANSWER_SEQ_OFFLINE;
-    uint256[] memory payload = new uint256[](2);
+    return _sendUpdateMessageToL2(currentAnswer);
+  }
+
+  /**
+   * @notice retries to send the latest answer as update message to L2
+   * @dev only with access, useful in cases where a previous x-domain message was handeled unsuccessfully.
+   */
+  function retry() external checkAccess returns (bool) {
```

Comment:
@cds95 I've changed this to `checkAccess` modifier here instead of `onlyOwner

I think this is more correct, as the owner could already add access to another party who could then retry by calling the `validate` fn manually. Here I suggest we lower the restriction to retry.

Let's say there is a use case to automate retries, this is also a required change to support those use cases.